src/carma-1-tenth/plot_multiple_crosstrack_errors.py

```diff
@@ -0,0 +1,73 @@
+# Plot the crosstrack error as a function of downtrack (distance traveled along the route)
```

Comment:
I'd just add some additional clarification that this is a box and whisker plot showing the crosstrack distribution or something along those lines that explains what the output looks like without generating the graph